src/contexts/globalLayout/GlobalAnnouncementPopover/GlobalAnnouncementPopover.tsx

```diff
@@ -0,0 +1,253 @@
+import Spacer from 'components/core/Spacer/Spacer';
+import { BaseM, TitleM } from 'components/core/Text/Text';
+import { useIsMobileOrMobileLargeWindowWidth } from 'hooks/useWindowSize';
+import { graphql, useFragment } from 'react-relay';
+import styled from 'styled-components';
+import { removeNullValues } from 'utils/removeNullValues';
+import { GlobalAnnouncementPopover$key } from '__generated__/GlobalAnnouncementPopover.graphql';
+import GalleryOfTheWeekCard from './Feed/GalleryOfTheWeekCard';
+
+type Props = {
+  queryRef: GlobalAnnouncementPopover$key;
+};
+
+// NOTE: in order to toggle whether the modal should appear for authenticated users only,
+// refer to `useGlobalAnnouncementPopover.tsx`
+export default function GlobalAnnouncementPopover({ queryRef }: Props) {
+  const query = useFragment(
+    graphql`
+      fragment GlobalAnnouncementPopover on Query {
+        ...GalleryOfTheWeekCardQueryFragment
+
+        galleryOfTheWeekWinners @required(action: LOG) {
+          dbid
+          ...GalleryOfTheWeekCardUserFragment
+        }
+      }
+    `,
+    queryRef
+  );
+
+  if (query === null) {
+    throw new Error('GlobalAnnouncementPopver did not receive gallery of the week winners');
+  }
+
+  const { galleryOfTheWeekWinners: rawData } = query;
+  const galleryOfTheWeekWinners = removeNullValues(rawData);
+
+  const isMobile = useIsMobileOrMobileLargeWindowWidth();
+
+  return (
+    <StyledGlobalAnnouncementPopover>
+      {
+        // A note on the architecture:
+        // Although desktop and mobile views are similar, they're sufficiently different such that they warrant
+        // different sets of components entirely. The alternative would be to litter this file with ternaries
+        // and CSS params
+        isMobile ? (
+          <>
+            <Spacer height={92} />
+            <MobileHeaderContainer>
+              <MobileIntroTextContainer>
+                <MobileIntroText>
+                  A new way to <i>connect</i> with collectors
+                </MobileIntroText>
+              </MobileIntroTextContainer>
+              <Spacer height={8} />
+              <MobileDescriptionTextContainer>
+                <BaseM>
+                  Collectors are looking for ways to connect with each other. To help them stay up
+                  to date with their favorite collectors on Gallery, we’re introducing our biggest
+                  change thus far—<b>a social feed</b>.
+                </BaseM>
+              </MobileDescriptionTextContainer>
+              <Spacer height={24} />
+              <MobileButtonContainer>
+                <StyledMobileButton text="Start Browsing" />
+                <Spacer height={16} />
+                <TextButton text="Galleries To Follow ↓" />
+              </MobileButtonContainer>
+            </MobileHeaderContainer>
+            <Spacer height={24} />
+            {/* TODO: replace this with recorded video */}
+            <img src="./temp-asset.jpg" />
```

Comment:
maybe you intend to do this, but leaving comment just in case - could be const since we're duplicating for desktop and mobile